backend/src/email/services/email.service.ts

```diff
@@ -154,17 +154,18 @@ const getCampaignMessage = async (
 const sendEmail = async (
   mail: MailToSend,
   opts?: SendEmailOpts
-): Promise<string | void> => {
+): Promise<boolean> => {
```

Comment:
It looked like all the callsites for `sendEmail` were only interested in using the return value (i.e. the SES message ID) as an indicator for whether the function succeeded or not. So I felt it was clearer just to return a `boolean` to indicate success. This more clearly communicates that we don't actually use the SES message ID.